templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+	<h1>{{ object|yesno:"Update, Create" }} Manufacturer</h1>
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {{ form|crispy }}
+    <input type="submit" value="Submit">
```

Comment:
It would be nice to have a Cancel button here that leads back to the previous page. This would improve the user experience as they could easily navigate back if they change their mind about creating or updating a Manufacturer.